components/system-broker/pkg/server/server.go

```diff
@@ -39,15 +39,18 @@ type Server struct {
 	shutdownTimeout time.Duration
 }
 
-func New(c *Config, service log.UUIDService, routesProvider ...func(router *mux.Router)) *Server {
+func New(c *Config, service log.UUIDService, middlewares []mux.MiddlewareFunc, routesProvider ...func(router *mux.Router)) *Server {
 	s := &Server{
 		shutdownTimeout: c.ShutdownTimeout,
 		routesProvider:  routesProvider,
 	}
 
 	router := mux.NewRouter()
+
+	router.Handle("/healthz", s.livenessHandler())
+	router.Handle("/readyz", s.readinessHandler())
```

Comment:
These should be behind the prefix in order for istio virtual service to work. You can just add the prefix to the probes.

Reply:
Discussed that all other components do not expose their health endpoints and are accessible only inside the cluster. This will be addressed everywhere if we decide to enhance our health checks.